sdk/appconfiguration/Azure.Data.AppConfiguration/src/autorest.md

```diff
@@ -0,0 +1,17 @@
+# Azure SDK Code Generation for Data Plane
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+``` yaml
+input-file:
+- https://github.com/Azure/azure-rest-api-specs/blob/e01d8afe9be7633ed36db014af16d47fec01f737/specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/1.0/appconfiguration.json
+namespace: Azure.Data.AppConfiguration
+title: ConfigurationClient
+public-clients: true
+directive:
+  from: swagger-document
+  where: $.parameters.Endpoint
+  transform: $.format = "url"
```

Comment:
Can we add a seperate directive for `Endpoint` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/anomalydetector/Azure.AI.AnomalyDetector/src/autorest.md#make-endpoint-type-as-uri) with a comment to make it more clear?